pyproject.toml

```
@@ -38,7 +38,7 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
 ]
 dependencies = [
-    "starlette==0.20.4",
+    "starlette>=0.21.0,<0.22.0",
```

Sebastian pins the patch as well.

```suggestion
    "starlette==0.21.0",
```